package.json

```diff
@@ -73,6 +80,45 @@
                 "language": "omt",
                 "path": "./snippets/omt.snippets.json"
             }
-        ]
+        ],
+        "configuration": {
+            "type": "object",
+            "title": "OMT-ODT configuration",
+            "properties": {
+                "languageServerExample.maxNumberOfProblems": {
+                    "scope": "resource",
+                    "type": "number",
+                    "default": 100,
+                    "description": "Controls the maximum number of problems produced by the server."
+                }
+            }
+        }
+    },
+    "main": "./client/out/extension",
+    "activationEvents": [
+        "onLanguage:omt",
+        "onLanguage:plaintext",
+        "onLanguage:typescript",
+        "onLanguage:json",
```

Comment:
Why do we want to activate the OMT/ODT plugin on plaintext, TypeScript and JSON files?